public/js/utils/source-map-worker.js

```diff
@@ -22,21 +33,21 @@ function enableSourceMaps() {
   sourceMapsEnabled = true;
 }
 
-function _resolveSourceMapURL(source) {
-  if (path.isURL(source.sourceMapURL) || !source.url) {
+function _resolveSourceMapURL(source: Source) {
+  const { url, sourceMapURL } = source;
```

Comment:
@jlongster I think if we should destructure the `source` object and set the `url` and `sourceMapURL` values to an empty string by default. It would pass Flow's static analysis for `path.isURL` or `path.isAbsolute` since they only expect a string here. /cc @jasonLaster